src/html-conventions-lib/class-html-conventions.xsl

```diff
@@ -215,7 +216,7 @@
 
  
 
-    <xd:doc>
+<!--    <xd:doc>
```

Comment:
as seen in the changes for this file this functionality was replaced by the new template elementUniqueName. It is commented out and not deleted just in case someone will want to revert the changes made in this pull request

Reply:
as seen in the changes for this file this functionality was replaced by the new template elementUniqueName. It is commented out and not deleted just in case someone will want to revert the changes made in this pull request

Reply:
deleted